client/gutenberg/extensions/vr/editor.js

```diff
@@ -0,0 +1,107 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { Placeholder, SelectControl, TextControl } from '@wordpress/components';
+import { registerBlockType } from '@wordpress/blocks';
+
+/**
+ * Internal dependencies
+ */
+import './editor.scss';
+
+registerBlockType( 'jetpack/vr', {
+	title: __( 'VR Image', 'jetpack' ),
+	description: __( 'Embed 360° photos and Virtual Reality (VR) Content', 'jetpack' ),
+	icon: 'embed-photo',
+	category: 'embed',
+	support: {
+		html: false,
+	},
+	attributes: {
+		url: {
+			type: 'string',
+			default: '',
+		},
+		view: {
+			type: 'string',
+			default: '',
+		},
+	},
```

Comment:
This one could also use some keywords. Maybe the `embed` from above could fit well here.

Reply:
Other suggestions could be `360`, `vr` or `panorama`